packages/dcos-integration-test/extra/test_dcos_checks.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import uuid
+
+
+def test_dcos_checks_detect_ip(dcos_api_session):
+    cmd = [
+        "/opt/mesosphere/bin/dcos-checks",
+        "--role",
+        "agent",
+        "components",
+        "--port",
+        "61001",
+        "--detect-ip",
+        "/etc/hosts",
+        "--exclude=dcos-checks-poststart.service,dcos-checks-poststart.timer"
+    ]
```

Comment:
Let's add a comment explaining that we expect this command to fail, and why we expect it to fail.

Reply:
same as @orsenthil comment, i added the explanation to test docstring